rifServices/src/main/java/org/sahsu/rif/services/datastorage/common/BaseSQLManager.java

```diff
@@ -284,6 +329,103 @@ else if (!allowNoRows) {
 		return columnValue;
 	}
 
+	/*
+	 * Fetch RIF view data for study/table
+	 *
+	 * @param: connection 		Connection,
+	 * @param: columnsAreString Boolean,
+	 * @param: columnName1 		String,
+	 * @param: columnValue1 	String,
+	 * @param: columnName2 		String,
+	 * @param: columnValue2 	String,
+	 * @param: tableName 		String,
+	 * @param: columnList 		String
+	 *
+	 * @returns: CachedRowSetImpl
+	 */
+	@Override
+	public CachedRowSetImpl getRifViewData(
```

Comment:
As above regarding the interface/implementation choice. Do we even need to be that specific here? We should probably just use `RowSet` where we can.

Reply:
It needs to be cached so the cursor can be rewound.